rp2-pio/statemachine.go

```diff
@@ -204,6 +196,18 @@ func (sm StateMachine) ClearFIFOs() {
 	xorBits(shiftctl, rp.PIO0_SM0_SHIFTCTRL_FJOIN_RX_Msk)
 }
 
+// GetRxFifoAt reads data from the RX FIFO at a specific index.
+// Requires FifoJoinRxPut mode to be enabled, RP2350-only.
+func (sm StateMachine) GetRxFifoAt(fifoIndex int) uint32 {
```

Comment:
In Go we generally leave acronyms as allcaps. See:
- https://go.dev/wiki/CodeReviewComments#initialisms
- Above methods i.e: `func (sm StateMachine) IsRxFIFOFull() bool {`